composer.json

```diff
@@ -65,6 +65,9 @@
             }
         }
     },
+    "conflict": {
+        "arcanedev/log-viewer": "^9.0" 
```

Comment:
it's possibly incompatible with other (earlier) versions of `arcanedev/log-viewer`, right? I think we should at least also include v8.0 here as well

Reply:
Fixed